sorbet/rbi/shims/bundler.rbi

```diff
@@ -1,6 +0,0 @@
-# typed: true
-
-class Bundler::StubSpecification
-  sig { returns(T::Boolean) }
-  def default_gem?; end
```

Comment:
Found by the feature itself: https://github.com/sorbet/sorbet/blob/master/rbi/stdlib/bundler.rbi#L9957-L9958